README.md

```diff
@@ -130,15 +134,15 @@ node_public_address = ""
 country = ""
 ```
 
-4. Create required directories
+1. Create required directories
 
 ```bash
 mkdir -p data logs
 ```
 
-5. Start the containers with the desired inference services
+1. Start the containers with the desired inference services, please note if you don't have a GPU, you'll need to use the you will need to use a `mistral` backend as currently the other vllms are not supported on CPU.
```

Comment:
We do have a `vllm-cpu` available container, but it only works with x86_64 archs, so for arm based machines we recommend using `mistralrs-cpu`.

Reply:
Thanks for the clarification. I've updated  in 5f595ea